decidim-core/app/views/decidim/participatory_processes/show.html.erb

```diff
@@ -12,4 +12,52 @@
       </div>
     </div>
   </div>
+  <% if participatory_process.attachments.any? %>
+    <div class="row attachments">
+      <div class="columns large-8">
+        <% if participatory_process.documents.any? %>
+          <div class="section documents">
+            <h4 class="section-heading"><%= t(".related_documents") %></h4>
+            <div class="card card--list">
+              <% participatory_process.documents.each do |document| %>
```

Comment:
Maybe it'd be worth to split this block into a partial to reduce indentation and complexity?